syntax = "proto3";

package jio.brain.proto.knowledge.api.data;

option java_package = "com.jio.brain.proto.knowledge.api.data";
option java_multiple_files = true;

import "jio/brain/proto/knowledge/base/value.proto"; 
import "jio/brain/proto/base/status.proto";

service GetAttributeValueService {
  rpc serve(GetAttributeValueRequest) returns (GetAttributeValueResponse) {}
}

message GetAttributeValueRequest {
  uint32 from_node_id = 1;
	uint32 to_node_id = 2;
  uint32 value_id = 3;
}

message GetAttributeValueResponse {
	jio.brain.proto.knowledge.base.Value value = 1;
  jio.brain.proto.base.BrainStatus status = 2;
}